packages/gatsby-remark-prismjs/src/highlight-code.js

```diff
@@ -10,7 +10,11 @@ module.exports = (language, code, lineNumbersHighlight = []) => {
       loadPrismLanguage(language)
     } catch (e) {
       // Language wasn't loaded so let's bail.
-      return code
+      if (_.includes([`plain`, `plaintext`, `text`], language)) {
+        return _.escape(code)
```

Comment:
Thanks! I'd noticed before that html in code samples sometimes would render as HTML which was confusing — why we didn't catch it before I think was that remark normally escapes code inside inline code and blocks so this problem doesn't happen. Also most code blocks do have languages set.